### PR TITLE
Code 142 resolce cc issues

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
### Jira Story

https://osi-cwds.atlassian.net/browse/CODE-142

## Description
Resolve Code Climate code smells and duplication for cognito-custom-login. The duplications were found in config files so included them in ignore files. And as for the max number of lines in loginpage.js the count was brought down to 286 from 307. I believe we ca leave it here as most of it doesn't include logic to complicate it.

Have not included any tests as the render methods that were created are already being tested in switch case tests.

4 Code climates issues resolved.

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -40,3 +40,4 @@ exclude_patterns:
   - "src/assets/"
   - "src/index.ejs"
   - "**/*.test.js"
+  - "*.config.*.js"
```
Comment (kelfink) [inline .codeclimate.yml]: new line here.  Or does YML complain if we have one?  I think it's safe

File: src/utils/validatePassword.js [hunk 2]
```diff
@@ -1,27 +1,7 @@
...
-    instance.setState({number: true})
-  } else {
-    instance.setState({number: false})
-  }
-  if (newPassword.match(/(\W)/gm)) {
-    instance.setState({specialCharacter: true})
-  } else {
-    instance.setState({specialCharacter: false})
-  }
+  newPassword.match(/.{8,}$/gm) ? instance.setState({maxLength: true}) : instance.setState({maxLength: false})
```
Comment (kelfink) [inline src/utils/validatePassword.js]:
```suggestion
  instance.setState({maxLength: newPassword.match(/.{8,}$/gm) })
```

Comment (kelfink) [reply src/utils/validatePassword.js]: same for the others

File: .codeclimate.yml (hunk 1 above)
Comment (NagaPradeepGajula-osi) [reply .codeclimate.yml]: the lint or the IDE doesn't complain about it. But, if needed I can add one.

**Pull request merged**